t80s_reduce/t80s_sort_night/teste.yaml

```diff
@@ -0,0 +1,83 @@
+night: '2016927'
```

Comment:
Another yaml configuration file that should not be part of the commit. Please, remove it.